app/assets/stylesheets/layout/_header.scss

```diff
@@ -19,11 +20,12 @@ header {
   a {
     color: $white;
     font-size: larger;
+    padding: 8px 0;
   }
 }
 
 @include desktop-lg {
-  header i {
-    display: none;
+  header {
+    flex-direction: row;
   }
 }
```

Comment:
Add an empty line at the end of your code

Reply:
done